tests/cypress/e2e/unit_tests/p1_fleet.spec.ts

```diff
@@ -316,3 +316,71 @@ if (!/\/2\.7/.test(Cypress.env('rancher_version'))) {
     )  
   });
 
+  describe('Test Self-Healing of IMMUTABLE resources modification when correctDrift option used for exisiting GitRepo', { tags: '@p1'}, () => {
```

Comment:
Suggestion to simplify title:

```suggestion
  describe(''Test Self-Healing on IMMUTABLE resources when correctDrift is enabled'', { tags: '@p1'}, () => {
```

Reply:
Done